src/main/java/com/epam/spring/homework3/service/repository/impl/HotelRepositoryImpl.java

```diff
@@ -0,0 +1,66 @@
+package com.epam.spring.homework3.service.repository.impl;
+
+import com.epam.spring.homework3.service.exception.NotFoundException;
+import com.epam.spring.homework3.service.model.Hotel;
+import com.epam.spring.homework3.service.repository.HotelRepository;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Component;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Component
```

Comment:
Use @Repository annotation instead of @Component, it has actually no difference, but first one has more readability to reviewers